lib/smart_proxy_dns_powerdns/dns_powerdns_main.rb

```diff
@@ -59,7 +59,7 @@ def create_ptr_record(fqdn, ptr)
 
     def do_create(name, value, type)
       zone = get_zone(name)
-      unless create_record(zone['id'], name, type, value) and rectify_zone(zone['name'])
+      unless create_record(zone['id'], name, type, value) and update_soa(zone['id'], zone['name']) and rectify_zone(zone['name'])
```

Comment:
I implements the SOA-EDIT. See https://doc.powerdns.com/md/authoritative/dnssec/#soa-edit-ensure-signature-freshness-on-slaves for the different ways it can increase the serial for you, but in short: if you've configured it correct then pdns will handle it for you.